lib/main.ts

```diff
@@ -26,11 +49,18 @@ export class Transpiler {
 
   translateProgram(program: ts.Program, relativeFileName: string): string {
     this.relativeFileName = relativeFileName;
-    return program.getSourceFiles()
-        .filter((sourceFile: ts.SourceFile) => !sourceFile.fileName.match(/\.d\.ts$/) &&
-                                               !!sourceFile.fileName.match(/\.[jt]s$/))
-        .map((f) => this.translate(f))
-        .join('\n');
+    var src = program.getSourceFiles()
+                  .filter((sourceFile: ts.SourceFile) => !sourceFile.fileName.match(/\.d\.ts$/) &&
+                                                         !!sourceFile.fileName.match(/\.[jt]s$/))
```

Comment:
This part is, it aligns the boolean clauses with the start of expression at `=>`. The source in createCompilerHost below sadly causes clang-format it to completely drop the ball. I'll investigate.